src/merkle/smt/mod.rs

```diff
@@ -79,28 +79,32 @@ pub(crate) trait SparseMerkleTree<const DEPTH: u8> {
     // PROVIDED METHODS
     // ---------------------------------------------------------------------------------------------
 
-    /// Returns an opening of the leaf associated with `key`. Conceptually, an opening is a Merkle
-    /// path to the leaf, as well as the leaf itself.
-    fn open(&self, key: &Self::Key) -> Self::Opening {
-        let leaf = self.get_leaf(key);
-
+    /// Returns a [`MerklePath`] to the specified key.
```

Comment:
nit: should this say "Returns a [`MerklePath`] to the leaf containing the specified key."